public/js/reducers/breakpoints.js

```diff
@@ -123,6 +123,9 @@ function update(state = State(), action: Action) {
       }
 
       break;
+    }
+    case "NAVIGATE": {
+      return State();
```

Comment:
That's a good point. I think this is actually a question of using a selector that only queries for breakpoints at that domain or breakpoints that have a source in the store, which can be difficult.